README.md

```diff
@@ -58,14 +58,16 @@ Example inspec.yml:
 name: my-profile
 title: My own Oneview profile
```

Comment:
Small nitpick. Would it be possible to update this to Azure profile?

Reply:
Yes please. (what's Oneview? and why is MS so bad at naming things?)